tuftool/src/root.rs

```diff
@@ -204,6 +212,13 @@ impl Command {
         write_file(path, &root)
     }
 
+    fn set_version(path: &PathBuf, version: NonZeroU64) -> Result<()> {
+        let mut root: Signed<Root> = load_file(path)?;
+        root.signed.version = version;
+        clear_sigs(&mut root);
```

Comment:
Do we need to clear signatures when we set a version ?

Reply:
As far as i know, whenever you modify root.json, we clear signatures.